.circleci/config.yml

```diff
@@ -1,8 +1,75 @@
+# with thanks to https://dev.to/jonatasbaldin/a-recipe-for-poetry-and-circleci-1bj6
 version: 2.1
 jobs:
-  build:
+  # Building and testing the project
+  # Useful when a PR is open, for example
+  build-and-test:
```

Comment:
if you make this `build` you won't have to change tf-github's expected status check

Reply:
looks like there's a handful of references to this to update though too

Reply:
fixed in 3198ba8, thanks for the catch!